src/background/context-menus.js

```diff
@@ -22,7 +22,7 @@ function handler(info, tab) {
         linkText = Markdown.imageFor("", info.srcUrl).markdown;
       } else {
         needEscape = true;
-        linkText = info.selectionText;
+        linkText = info.selectionText ? info.selectionText : info.linkText;
```

Comment:
Good to know that this is available from Firefox 56. https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/menus/OnClickData

To transfer the knowledge, please add a `// NOTE` about why there are two different properties we can use, and browser compatibility issue.